microsoft-edge/extensions-chromium/developer-guide/minimize-page-load-time-impact.md

```diff
@@ -0,0 +1,251 @@
+
+1. To start recording a performance profile, click **Record** (![Record icon](./minimize-page-load-time-impact-images/record-icon.png)).
+
+1. Reload the page to capture the profiling data that corresponds to the page load time and then, once the page has finished loading, click **Stop** (![Stop icon](./minimize-page-load-time-impact-images/stop-icon.png)) to end the recording. DevTools displays the recorded performance profile.
+
+   ![The recorded performance profile displayed in the DevTools Performance tool](./minimize-page-load-time-impact-images/profile.png)
+
+1. To search for the performance events that are caused by your content script, press **Ctrl+F** on Windows/Linux or **Command+F** on macOS.  The **Find** text box appears at the bottom of the **Performance** tool.
+
+1. Type **Evaluate script** and then press **Enter** until the **Performance** tool highlights the performance events that are caused by your content script.  You will know that you have found the right performance event when the **Script**  label in the **Summary** panel shows the name of your content script.
+
+   ![The Evaluate script performance event due to the extension's content script running during the page load](./minimize-page-load-time-impact-images/evaluate-script-event.png)
+
+
+<!-- ------------------------------ -->
+#### Profile your content script using the Edge tracing tool
+
+The Edge tracing tool, available at the `edge://tracing` URL, is a powerful tool that can provide detailed analysis of your extension's performance. In this section, you'll learn how to use the Edge tracing tool to understand the impact of your extension on the page load time. To learn more about the Edge tracing tool, which is based on the **Perfetto** tool, see [Perfetto UI](https://perfetto.dev/docs/visualization/perfetto-ui).
+
+1. To open the Edge tracing tool, open a new tab or window, and go to `edge://tracing`. The tracing UI opens.
+
+1. To start a new trace, in the top left corner of the tool, click **Record**. The **Record a new trace** dialog box appears.
+
+1. In the **Record a new trace** dialog box, select the **Manually select settings** option. The list of categories appears.
+
+1. To capture detailed information about your extension's content script compilation and execution, select the following categories:
+
+   * **extensions**
+   * **v8**
+   * **devtools**
+   * **devtools.timeline**
+
+1. Click **Record**. The Edge tracing tool starts recording the trace.
+
+1. Open a new tab and load a webpage that your extension affects. The tracing tool collects data about the performance impact of your extension on the webpage.
+
+1. Open the tab where the Edge tracing tool is running, and then click **Stop**. The new trace information appears in the tool.
```

Comment:
```suggestion
1. Open the tab where the Edge tracing tool is running, and then click the **Stop** (![icon png]) button. The new trace information appears in the tool.
```
if approp, add icon png

Reply:
found existing icon png & tagging:
click the **Stop** (![Stop icon](./minimize-page-load-time-impact-images/stop-icon.png)) button 
will edit the above comment to add that in Suggestion tagging

Reply:
no, it's a rect button not icon button